optimize_epanet.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+from scipy.optimize import differential_evolution
+from epanet_end_error import run_epanet_and_calculate_error
+
+
+junctions_to_adjust = [' SO/ZD', ' SW/SO', ' SW/RO', ' SD/GO1', ' SW/AN', ' HP7']
+pipes_to_adjust = [' p23', ' p129', ' p159']
+
```

Comment:
Brak linii
Zgodnie z konwencją (PEP 8) powinny być 2 przed funkcją
// tak wiem, że straszne czepialstwo, ale potem na rozmowach o pracę się o to też czepiają